sql/core/src/test/scala/org/apache/spark/sql/CachedTableSuite.scala

```diff
@@ -634,4 +634,28 @@ class CachedTableSuite extends QueryTest with SQLTestUtils with SharedSQLContext
       assert(getNumInMemoryRelations(cachedPlan2) == 4)
     }
   }
+
+  test("refreshByPath should refresh all cached plans with the specified path") {
+    def f(path: String, spark: SparkSession, dataCount: Int): DataFrame = {
+      spark.catalog.refreshByPath(path)
```

Comment:
we can put `spark.range(dataCount).write.mode("overwrite").parquet(path)` at the beginning of this method and name it `testRefreshByPath` instead of `f`